### PR TITLE
Part 145,146 commit Issue 78, Issue 62

### PR DESCRIPTION
**Pull request by SRS-WRKS**
Part 145 of commit on Issue #78 and Issue #62
Part 146 of commit on Issue #78 and Issue #62

### CODE REVIEW

File: concepts/snobol3.pldb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 title SNOBOL3
-appeared 1996
+appeared 1966
```
Comment (breck7) [inline concepts/snobol3.pldb:2]: nice fix!

File: concepts/sophia.pldb [hunk 2]
```diff
@@ -3,6 +3,8 @@ appeared 2018
 type contractLanguage
 creators Robert Virding
 website https://aeternity.com/aesophia
+country Liechtenstein
```
Comment (breck7) [inline concepts/sophia.pldb:6]: wow! the first PL from Liechtenstein :)

**Comment by breck7** [comment]
Great work @SRS-WRKS !

**Comment by SRS-WRKS** [comment]
@breck7 : You review my patches after a long time :) .. feels like old times

**Pull request merged**